react/features/connection-indicator/components/ConnectionIndicator.js

```diff
@@ -430,6 +433,7 @@ class ConnectionIndicator extends Component {
                 packetLoss = { packetLoss }
                 region = { region }
                 resolution = { resolution }
+                serverRegion = { serverRegion }
                 shouldShowMore = { this.state.showMoreStats }
```

Comment:
Dang, maybe at this point, not saying for this PR, ConnectionStatsTable should just take in the stats object itself or we use a spread operator to pass in all stat values (`{ ...this.state.stats }`)or have ConnectionStatsTable listen instead of just ConnectionIndicator.